main.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+
+from indexclient.client import IndexClient
+from cdislogging import get_logger
+
+from settings import INDEXD
+from utils import get_fileinfos_from_tsv_manifest
+
+logger = get_logger(__name__)
+
+def manifest_indexing(manifest, prefix=None):
+    """
+    Loop through all the files in the manifest, update/create records in indexd
+    update indexd if the url is not in the record url list or acl has changed
+
+    """
+    indexclient = IndexClient(
+        INDEXD["host"],
+        INDEXD["version"],
+        (INDEXD["auth"]["username"], INDEXD["auth"]["password"]),
+    )
+    try:
+        files = get_fileinfos_from_tsv_manifest(manifest)
+    except Exception as e:
+        logger.error("Can not read {}. Detail {}".format(manifest, e))
+        return
+
+    prefix = prefix + "/" if prefix else ""
+    number_indexed_files = 0
+    for fi in files:
+        try:
+            url = fi.get("url")
+
+            if fi.get("acl") in {"[u'open']", "['open']"}:
+                acl = ["*"]
+            else:
+                acl = [element.strip() for element in fi.get("acl")[1:-1].split(",")]
```

Comment:
if acl in the file is `['a']` you will get `["'a'"]` and i think we want `["a"]`?